tests/drivers/can/api/src/main.c

```diff
@@ -968,12 +968,15 @@ static void test_filters_preserved_through_mode_change(void)
 	can_remove_rx_filter(can_dev, filter_id);
 }
 
-static void test_filters_preserved_through_bitrate_change(void)
+ZTEST_USER(can_api_test, test_filters_preserved_through_bitrate_change)
 {
 	struct zcan_frame frame;
 	int filter_id;
 	int err;
 
+	err = can_set_mode(can_dev, CAN_MODE_LOOPBACK);
+	zassert_equal(err, 0, "failed to set loopback-mode (err %d)", err);
+
```

Comment:
How come you are adding this?

Reply:
this case run be before test_set_loopback() and failed, so I copy the statements in test_set_loopback() to this function. Now the test_set_loopback() is part of setup function, this two line are removed.